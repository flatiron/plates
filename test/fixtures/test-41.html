<form class="ssl">
  <fieldset>
    <input type="text">
    <textarea></textarea>
  </fieldset>
  <fieldset>
    <input type="text">
    <textarea></textarea>
  </fieldset>
  <fieldset>
    <input type="text">
    <textarea></textarea>
  </fieldset>
</form>
<form>
  <fieldset>
    <input type="text">
    <textarea></textarea>
  </fieldset>
</form>
