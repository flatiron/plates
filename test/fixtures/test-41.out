
<form>
  <fieldset>
    <input type="text">
    <textarea></textarea>
  </fieldset>
</form>
